### PR TITLE
[UX][Web][SIMS #1451] UI Content for Institution - Profile page (Part 3 of 3)

### PR DESCRIPTION
**Pull request by hellolynn-tbtb**
### Profile page (Part 3 of 3)
- Content update in banner
- Content update in form
- CSS enhancements - aligned disabled fields styles that weren't inheriting "grey bg" or "cursor disabled" style

### Additional information
- [Figma mockups (for reference only)](https://www.figma.com/file/9Om2tzLAVEoMQrjmrH4fhu/MOD-SIMS?node-id=3651%3A95172)
- Added small update on unrelated page & task - institution login page, nav re-direct to profile instead of locations
- To create another PR for word wrap FIX (likely float issue)
image

### Screenshot
screencapture-localhost-8081-institution-institution-profile-edit-2022-11-04-15_07_14

### CODE REVIEW

File: sources/packages/web/src/views/institution/Login.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
             Manage your institution account using your Business or Basic BCeID
             User ID. Learn how to setup and access your account
             <a
-              class="primary-color"
+              class="primary-color formio-href"
```
Comment (ann-aot) [inline sources/packages/web/src/views/institution/Login.vue:13]: `formio-href` has `color = $brand-blue;`. so we can remove `primary-color` class

File: sources/packages/web/src/views/institution/AppInstitution.vue [hunk 2]
```diff
@@ -21,7 +21,7 @@
           v-if="isAuthenticated && isAdmin"
           variant="text"
           :to="{
-            name: InstitutionRoutesConst.MANAGE_LOCATIONS,
+            name: InstitutionRoutesConst.INSTITUTION_PROFILE_EDIT,
```
Comment (ann-aot) [inline sources/packages/web/src/views/institution/AppInstitution.vue:24]: 👍

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

File: sources/packages/web/src/assets/css/global-style-variables.scss [hunk 3]
```diff
@@ -18,7 +18,7 @@ $banner-background-info: #f0eef9;
 /** Background and font colors - Figma **/
 $common-border-color: #e8e8e8;
 $color-white: #ffffff;
-$background-read-only: #cfd4db;
+$read-only-background: #e9ecef;
```
Comment (ann-aot) [inline sources/packages/web/src/assets/css/global-style-variables.scss:21]: do we need to update vuetify theme `readonly`?
image
so, these are used by the vuetify components

File: sources/packages/web/src/views/institution/Login.vue (hunk 1 above)
Comment (hellolynn-tbtb) [reply sources/packages/web/src/views/institution/Login.vue:13]: Thanks ann - good point!

File: sources/packages/web/src/assets/css/global-style-variables.scss (hunk 3 above)
Comment (hellolynn-tbtb) [reply sources/packages/web/src/assets/css/global-style-variables.scss:21]: @ann-aot  Oh yes we should - thank you for noting that! I changed it now.

**Review by ann-aot — APPROVED** [review]
Looks good @hellolynn-tbtb 👍

**Pull request merged**